mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule SituationRoom.MixProject do
     [
       app: :situation_room,
       version: "0.1.0",
-      elixir: "~> 1.13.4",
```

Comment:
Why the downgrade on version here?  Also, we don't need to specify the patch version here to avoid this.  Can just be `"~> 1.13"

Reply:
I downgraded due to the CI image. There is no official circle CI image for elixir 1.13.4. I figured it would not be a big deal downgrading for that reason. I think your suggestion to make the version 1.13 would also work just fine.